grafast/grafserv/package.json

```diff
@@ -115,6 +115,7 @@
     "@types/koa": "^2.13.8",
     "@types/koa-bodyparser": "^4.3.10",
     "@whatwg-node/fetch": "^0.9.10",
+    "@whatwg-node/server": "^0.9.64",
```

Comment:
Looking at the code, this should also be an optional peer dependency, otherwise `createServerAdaptor` cannot be imported? (The other adaptors do not require anything but types be imported.)

Reply:
Done